pandas/_libs/tslibs/period.pyx

```diff
@@ -322,6 +376,19 @@ cdef int dInfoCalc_SetFromAbsDate(date_info *dinfo, int64_t absdate) nogil:
 cdef int dInfoCalc_SetFromAbsTime(date_info *dinfo, double abstime) nogil:
     """
     Sets the time part of the DateTime object.
+
+    Parameters
+    ----------
+    dinfo : date_info*
+    abstime : double
+
+    Returns
+    -------
+    code : int (always 0)
```

Comment:
success / error

Reply:
It's actually a dummy; there is no error-state.  I guess just make it return None?